sdk/core/Azure.Core/src/Cryptography/IKeyEncryptionKey.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Core.Cryptography
```

Comment:
So we'll use this from Azure.Storage.Blobs.Cryptography and you'll implement it in Azure.Security.KeyVault.Cryptography?  That's a good namespace choice then.